packages/auth/src/app.ts

```diff
@@ -111,11 +114,9 @@ export class App {
       ctx.status = 200
     })
 
+    const grantRoutes = await this.container.use('grantRoutes')
     // TODO: GNAP endpoints
-    this.publicRouter.post('/auth', (ctx: AppContext): void => {
-      // TODO: generate interaction session
-      ctx.status = 200
-    })
+    this.publicRouter.post('/auth', grantRoutes.auth.post)
```

Comment:
I know the OpenAPI spec currently has `/auth` on the base server url
https://github.com/interledger/open-payments/blob/master/auth-server-open-api-spec.yaml#L13
but I don't think the `/auth` path is required by GNAP.
What do you think about changing this to
```suggestion
    this.publicRouter.post('/', grantRoutes.create)
```
(`create` to distinguish from the `POST /continue` route method)

Reply:
That works for me. I take it removing the `/auth/` path applies to all the other routes?